tensorflow/lite/micro/tools/ci_build/test_x86.sh

```diff
@@ -54,3 +54,14 @@ readable_run make -s -j8 -f tensorflow/lite/micro/tools/make/Makefile build
 readable_run make -s -j8 -f tensorflow/lite/micro/tools/make/Makefile test
 readable_run make -s -j8 -f tensorflow/lite/micro/tools/make/Makefile integration_tests
 
+# Next clean up everything
+readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
+
+# At last test the hello_world as an example outside of the github repo.
```

Comment:
nit: let's reorder this a bit so that the make clean is part of the testing block for the external dir.

```bash
# Test the ability to build an example outside of the TFLM tree.
readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
cd ../
```

Reply:
Done.